quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -200,7 +200,7 @@ impl PostgresqlMetastore {
 
     /// Publish splits.
     /// Returns the successful split IDs.
-    fn publish_splits(
+    fn mark_splits_as_published(
```

Comment:
I see, you consistently renamed the private functions `mark_splits_as_published`.